flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+  "only-arches": ["x86_64", "aarch64"],
+  "skip-icons-check": true,
+  "disable-external-data-checker": true
```

Comment:
why not using the built-in one?

Reply:
This is done to be "future-proof" regarding a future release of the runtime `23.08`, matching what has been done for the .NET 6 SDK with the `21.08` and `22.08` runtimes. For reference, [here is the PR] (https://github.com/flathub/org.freedesktop.Sdk.Extension.dotnet6/pull/32) which describes why this workaround is used to keep both branches updated
Of course, it isn't a problem currently, since it isn't out yet, I just wanted to set it once and forget it to be completely honest.